Program.cs

```diff
@@ -1,7 +1,29 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+using System;
+internal class Program {
+   private static void Main (string[] args) {
+      int a = 1, z = 100, attempt = 1, guess;
+      Console.Write ($"Guess random number from {a} to {z+1} and press enter key.");
+      Console.ReadKey ();
+      while (true) {

```

Comment:
If user uses No as answer, the output keep on repeating "Is that lesser than 100? Y/N: AND
Is 100 your answer?(Y/N):". 
Can you think and set maximum number of iteration before telling user "Sorry I failed to find your out-of-the-box guessing!"

Reply:
Repetition of questions reduced.